yt_astro_analysis/radmc3d_export/RadMC3DInterface.py

```diff
@@ -232,10 +232,10 @@ def write_amr_grid(self):
             ix = int(ind[0]+0.5)
             iy = int(ind[1]+0.5)
             iz = int(ind[2]+0.5)
+            nx, ny, nz = layer.ActiveDimensions / 2
```

Comment:
Would it be possible to do `layer.ActiveDimensions // 2` ?  I think that might eliminate the need to cast to `int` at all.

Reply:
Yes, that would be better. Thanks a lot!

Reply:
Sorry, I noticed that the type of nx, ny, nz won't change if only change the code to "layer.ActiveDimensions // 2".
It remains "float", so maybe it need another method.

Reply:
Yes, evidently `//` will return a float if either the numerator or denominator is a float. I was surprised by this, too.